HLTrigger/HLTfilters/plugins/L1TTkEmFilter.cc

```diff
@@ -147,7 +152,8 @@ bool L1TTkEmFilter::hltFilter(edm::Event& iEvent,
         passIsolation = true;
     }
 
-    if (offlinePt >= min_Pt_ && itkEm->eta() <= max_Eta_ && itkEm->eta() >= min_Eta_ && passQuality && passIsolation) {
+    if (offlinePt >= min_Pt_ && std::fabs(itkEm->eta()) < max_AbsEta1_ && std::fabs(itkEm->eta()) >= min_AbsEta1_ &&
```

Comment:
elsewhere in this file `std::abs` is used

https://github.com/cms-sw/cmssw/blob/a96c90c81f11a669e86a6da78e3df0f69a684bbb/HLTrigger/HLTfilters/plugins/L1TTkEmFilter.cc#L181-L182

I think that's the preferred option.